objection/console/cli.py

```diff
@@ -255,18 +255,25 @@ def patchipa(source: str, gadget_version: str, codesign_signature: str, provisio
                                               'specified, the latest version will be used.'), default=None)
 @click.option('--pause', '-p', is_flag=True, help='Pause the patcher before rebuilding the APK.',
               show_default=True)
-@click.option('--skip-cleanup', '-k', is_flag=True,
+@click.option('--skip-cleanup', '-k', is_flag=True, default=True,
```

Comment:
Not sure to understand this comment (and the following ones) about switching to flags. Isn't this the current way the code is written in `master` branch? https://github.com/sensepost/objection/blob/master/objection/console/cli.py#L270-L271

Reply:
@Phyks you are correct that they are currently flags, it's not that they need to be switched to flags. Leonza was saying that changing the default to true would change the option's meaning. Simply, if you were to specify "--skip-cleanup" it would be performing cleanup instead of skipping it (since skipping it would be the default). 
With that in mind, he wished to keep this option as it, since it would be better to perform cleanup by default.

Reply:
Reverted as well